packages/components/src/modal/style.scss

```diff
@@ -88,6 +102,19 @@
 	}
 }
 
+// Note: this keyframe definition is also used in the animationend JS event
+// listener — make sure the two instances are kept in sync.
```

Comment:
👍 

Should we clarify we're talking about keeping the animation name in sync? Currently it's not clear if we're talking about the animation name or the actual animated properties inside the keyframes.